app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/exemptionandlosses/annual_exempt_amount.scala.html

```diff
@@ -112,7 +113,7 @@
           @details(messages(s"$key.details.1.header"), Html(messages(s"$key.details.1.body", appConfig.trustsTaxFreeAllowance)))
           @details(messages(s"$key.details.2.header"),  Html(messages(s"$key.details.2.body", appConfig.trustsForVulnerable)))
         } else {
-          <p>@Html(messages(s"$key.link", appConfig.annualExemptAmountUrl))</p>
+          @details(messages(s"$key.details.1.header"),  Html(messages(s"$key.details.1.body", appConfig.annualExemptAmountUrl)))
```

Comment:
should the non-trust, non-period of admin stuff have changed here as well?

Reply:
refined the logic to only apply this for period of admin